src/components/controls/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -84,6 +69,65 @@ describe('TextArea', () => {
         });
     });
 
+    describe('error', () => {
+        test('render error message with error prop', () => {
+            const {container} = render(<TextArea error="Some Error" />);
+
+            // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
```

Comment:
Let`s rewrite this with [this](https://testing-library.com/docs/queries/bytext/) helper